src/DDNSUpdate/Infrastructure/Hosting/ReturnCode.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace DDNSUpdate.Infrastructure.Hosting
+{
+    public static class ReturnCode
```

Comment:
Prefer enums

Reply:
> 
> 
> Prefer enums

It's used as a returncode in program.cs, so we'd have to cast it, if it was an enum.